packages/sui-i18n/README.md

```diff
@@ -267,32 +361,215 @@ i18n.url('/HOME') //=> 'www.example.com/home'
 i18n.url('/HOME/REDIRECT_TRUE', true) //=> ''www.example.com/home/?redirect=true'
 i18n.url('/HOME/REDIRECT_TRUE') //=> 'www.example.com/home/redirecttrue'
 i18n.url('/HOME/REDIRECT_TRUE', false) //=> 'www.example.com/home/redirecttrue'
-
 ```
 
 ### Use with ReactJS
 
 ```javascript
 // index.js
 
-import I18n from '@s-ui/i18n';
-import Polyglot from '@s-ui/i18n/lib/adapters/polyglot';
+import I18n from '@s-ui/i18n'
+import Polyglot from '@s-ui/i18n/lib/adapters/polyglot'
 import withContext from '@s-ui/hoc/lib/withContext'
-import App from '../app';
+import App from '../app'
 
-const i18n = new I18n({adapter: new Polyglot()});
+const i18n = new I18n({adapter: new Polyglot()})
 const AppWithContext = withContext({i18n})(App)
-React.render(
-    <AppWithContext />,
-    document.getElementById('app')
-);
+React.render(<AppWithContext />, document.getElementById('app'))
+```
+
+### Usage Best practices
+
+While translating and interpolating text, we could face different scenarios where using a method is a better choice compared to others.
+
+As a golden rule, you'd prefer to always use the **i18n.t** method for translation, unless you have styles or transformations nested inside the text that requires a custom interpolation to avoid creating multiple separated strings.
+
+Here a list of the most common use cases and the related best practice:
+
+#### Simple text: **i18n.t** 🏆
+
+```jsx
+import I18n from '@s-ui/i18n'
+import Polyglot from '@s-ui/i18n/lib/adapters/polyglot'
+
+const i18n = new I18n({adapter: new Polyglot()})
+i18n.languages = {
+  'en-GB': {
+    TEXT_KEY: 'Complete your data!'
+  }
+}
+
+// ✅ Do
+function Component() {
+  const label = i18n.t(`TEXT_KEY`)
```

Comment:
Instead of using ` here maybe would be better to use '. e.g. 'TEXT_KEY'

Reply:
It's a leftover from other examples, good catch, let me fix it :)